.github/stale.yml

```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - do-not-stale
+  - security
+# Label to use when marking an issue as stale
+staleLabel: need-more-info
```

Comment:
Does this mark it as stale immediately?

Reply:
Or does it add it WHEN it goes stale? We can add to the comment, but if it's added when it goes stale, then I think the label `gone-stale` or similar makes sense. Issues could go stale because of us, not because of the reporter, in which case `need-more-info` might not be correct.

Reply:
Although I think that we can extend the deadlines, I don't agree with your premise: the whole logic behind this bot is to clean up issues that are idle for way too long - we have issues from 2016, which clearly means no one will act on them so we might as well close it and re-open if/when the motivation/need comes back for us or an user.

And if we intend to work on an issue, that's why the `do-not-stale` is here for (I'd go even further as to suggest that if one of adds such tag, he/she becomes the assignee for the issue - but that's another topic). There is no point keeping indefinitely issues open in the idea that *someone* may work on it in a possible feature.

Reply:
> Does this mark it as stale immediately?

I think so, but I can't test it out on a mock repo because I can't install the `stalebot` without providing GitHub with payment details (which I don't want) even if it costs 0$. If one of you has the possibility to test it that would probably avoid confusion later on.

> we have issues from 2016, which clearly means no one will act on them so we might as well close it and re-open if/when the motivation/need comes back for us or an user.

> There is no point keeping indefinitely issues open in the idea that someone may work on it in a possible feature.

I think closing old issues that are not being worked on is fine as long as they are marked somehow as `stale` or `unresolved` or something so they can be found easily among the closed and resolved issues in case someone does finally want to work on them. But I do agree with @Grazfather that a `need-more-info` label can be confusing if actually every necessary information has been provided already: I would not click on that issue when I have time to spare and want to work on an old issue because I would think that there is information missing.

So if **every** stale issue/pr gets labeled `need-more-info` than the label should be changed to `stale`/`gone-stale` or something. But if **just** issues/prs labeled with `need-more-info` become `stale` immediately that would be even better (but then an additional `gone-stale` label might be good for the other issues that have provided enough info).

I hope my wording is not too confusing^^